working/augmentation-libraries/feature-specification.md

```diff
@@ -1286,13 +960,13 @@ For example, we define the *augmented parameter list* of a non-empty stack,
 _This will usually be exactly the parameter list of the introductory
 declaration, but the ordering of named parameters may differ. This is mostly
 intended as an example, in practice the augmented parameter list can just be
-the parameter list of the introductory declaration, but it’s more
+the parameter list of the introductory declaration, but it's more
 direct and clearly correct to use the actual parameter list of the declaration
 when creating the parameter scope that its body will run in._
 
 Similarly we define the _augmented function type_ of the declaration stack.
 Because of the restrictions we place on augmentations, they will all have the
-same function type as the introductory declaration, but again it’s
+same function type as the introductory declaration, but again it's
 simpler to assign a function type to every declaration.
```

Comment:
Presumably, this step should gather metadata as well?

Reply:
@lrhn pointed out in his review that metadata doesn't actually have semantics in the language. So I added a section basically saying it's up to tools to decide how metadata is handled but suggesting that it's probably most useful to append it.